### PR TITLE
Fix assembly references in tests

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: src/coreclr/tests/src/Loader/classloader/DefaultInterfaceMethods/reabstraction/reabstraction.il [hunk 1]
```diff
@@ -4,6 +4,7 @@
 //
 
 .assembly extern mscorlib { }
+.assembly extern System.Private.CoreLib { }
```
Comment (jkotas) [inline src/coreclr/tests/src/Loader/classloader/DefaultInterfaceMethods/reabstraction/reabstraction.il]: This should reference System.Runtime instead. Ideally, none of the tests would reference System.Private.CoreLib directly.

File: src/coreclr/tests/src/JIT/Directed/Convert/ldind_conv.il [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 
 .assembly extern mscorlib { auto }
-.assembly extern System.Private.CoreLib { auto }
+.assembly extern System.Runtime { auto }
 
 .assembly test { }
```
Comment (jkotas) [inline src/coreclr/tests/src/JIT/Directed/Convert/ldind_conv.il]: `.class auto Program extends [System.Private.CoreLib]System.Object` below needs to be changed to `System.Runtime` as well. (Multiple places.)

Comment (NextTurn) [reply src/coreclr/tests/src/JIT/Directed/Convert/ldind_conv.il]: How to deal with this `call void [System.Private.CoreLib]Internal.Console::WriteLine`?

Comment (jkotas) [reply src/coreclr/tests/src/JIT/Directed/Convert/ldind_conv.il]: Change it to `[System.Console]System.Console.WriteLine` or `[mscorlib]System.Console.WriteLine`

**Review by jkotas — APPROVED** [review]
Thank you!

**Pull request merged**